src/main/navigation/navigation_pos_estimator_agl.c

```diff
@@ -150,7 +150,7 @@ void estimationCalculateAGL(estimationContext_t * ctx)
         const float accWeight = navGetAccelerometerWeight();
         posEstimator.est.aglAlt += posEstimator.est.aglVel * ctx->dt;
         posEstimator.est.aglAlt += posEstimator.imu.accelNEU.z * sq(ctx->dt) / 2.0f * accWeight;
-        posEstimator.est.aglVel += posEstimator.imu.accelNEU.z * ctx->dt * sq(accWeight);
```

Comment:
This feels wrong to me. In AGL mode, the sensor that measures distance to ground should have higher gains than accelerometer. This feels like it was done like this on purpose.

Reply:
Let me explain.
By the time they wrote this code (5 years ago), the way the accelerometer weight worked was different.
https://github.com/iNavFlight/inav/blob/e64753b46e0d3906c78b54bd7e09b14e2bb8cf6c/src/main/navigation/navigation_pos_estimator.c#L483-L509
Originally, its value was fixed and there was a setting for that. Therefore, the weight was squared for vel estimations. But when they changed the way the accelerometer weight works, forgot to update this file. Now the problem is that the `accWeight` can go down to 0.3 automatically and squared is `0.3*0.3=0.09` which is too low. Some rangefinders like the one in the mtf-01 is too slow taking measurements which makes the navigation very unstable.